tests/rados/stretch_cluster.py

```diff
@@ -349,8 +421,10 @@ def set_osd_sites(
     Returns: True -> pass, False -> fail
     """
     # adding the identified OSD's into the respective sites
-    if site != 1 or site != 2:
-        log.error("Site Values can only be either 1 or 2")
+    sites = set()
+    sites.add(site)
+    if len(sites) > 2:
+        log.error("There can only be 2 Sites with stretch cluster at present")
```

Comment:
The Site names are not restricted to any integer or any character. It is just setting two unique strings for names of site. Hence i opted this method. checking the int does not work here.

Reply:
https://github.com/red-hat-storage/cephci/pull/572/files/f38a6c63fa521ecac5e35440b78506a5580752ca#diff-5ad046aa282c1411656b54d58c19850af0e671f9370a099926da8db898fa0b92R409 

https://github.com/red-hat-storage/cephci/pull/572/files/f38a6c63fa521ecac5e35440b78506a5580752ca#diff-5ad046aa282c1411656b54d58c19850af0e671f9370a099926da8db898fa0b92R416

The method signature and documentation indicates otherwise. If this PR, introduces a change in this area... kindly update them also.

Reply:
Updated the docstring.